charts/chronicle/templates/tests/auth-endpoints-test.yaml

```
@@ -0,0 +1,42 @@
+{{- if .Values.test.auth.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-auth-endpoints-test
+  labels: {{ include "chronicle.labels" . | nindent 4 }}
+    component: auth-endpoints-test
+  annotations:
+    "helm.sh/hook": test
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName:  {{ include "lib.serviceAccountName" . }}
+      containers:
+        - name: curl-jwks
```

```suggestion
        - name: curl-oidc
```
It's testing both JWKS and userinfo.